src/gt4py/ir/nodes.py

```diff
@@ -132,6 +135,7 @@
                       domain: Domain,
                       api_fields: List[FieldDecl],
                       parameters: List[VarDecl],
+                      splitters: List[VarDecl],
```

Comment:
If we impose an ordering on the splitters in the same way the vertical does, then I think it can be compatible (with the future run-time vertical splitters). However, currently the splitters are not ordered, since my concern was an extra run-time check needed to ensure the run-time values passed are ordered (more stencil overhead). I did not think ordering was necessary in the horizontal, since there is (currently) no reordering/scheduling of the blocks in the way it is necessary to do so in the vertical. Do you think they should still be ordered?

If we leave them unordered, I could see renaming these `parallel_splitters` or `varsplitters`. Are there any others you can think of?